perceval/simulators/simulator_interface.py

```diff
@@ -124,7 +124,7 @@ def probs(self, input_state) -> BSDistribution:
         results, _ = self._postprocess_bsd(results)
         return results
 
-    def probs_svd(self, svd: SVDistribution, progress_callback: callable = None) -> dict:
+    def probs_svd(self, svd: SVDistribution, progress_callback: callable = None, detectors = None) -> dict:
         probs = self._simulator.probs_svd(self._prepare_input(svd), progress_callback)
```

Comment:
Missing the detectors transmission here?

Reply:
Will be resolved in PCVL-819